ui/src/api/index.ts

```diff
@@ -28,7 +28,16 @@ globalAxios.interceptors.response.use(
       );
       location.href = '/#/login';
     } else {
-      message.error(error?.response?.data?.message || error.message);
+      const { response } = error;
+      if (
+        response?.status === 400 &&
+        response?.config?.url?.split('/')?.pop() === 'password' &&
+        response?.data?.message === 'Error BadRequest: password is incorrect'
+      ) {
+        message.error('原密码输入不正确')
```

Comment:
The specific judgement for incorrect password here is not so proper. Maybe putting them in another file named like `errorHandling.ts` is better.